resources/models/resource.py

```diff
@@ -378,3 +392,33 @@ class Meta:
         verbose_name = _('resource image')
         verbose_name_plural = _('resource images')
         unique_together = (('resource', 'sort_order'),)
+
+
+class ResourceEquipment(ModifiableModel):
+    resource = models.ForeignKey(Resource, related_name='resource_equipments')
+    equipment = models.ForeignKey(Equipment, related_name='resource_equipments')
+    data = hstore.DictionaryField(null=True, blank=True)
```

Comment:
So -- and I guess this is the most controversial point of the CR -- uh, what's the use case for making this a hstore field? What sort of `data` will be storing here in the near future that couldn't be normalized as normal relational fields or stored in a regular [`JSONField` (that naturally can't be directly queried)](https://github.com/bradjasper/django-jsonfield).

Now to be clear, I'm not directly criticizing this technical decision, I'd just like to know the reasoning.